compiler/compile/OMRCompilation.cpp

```diff
@@ -2732,3 +2732,25 @@ OMR::Compilation::getSnippetsToBePatchedOnClassRedefinition()
    {
    return self()->cg()->getSnippetsToBePatchedOnClassRedefinition();
    }
+
+TR::Block *
+OMR::Compilation::insertNewFirstBlock()
+   {
+   TR::Node *oldBBStart = getStartTree()->getNode();
+   TR::Block *oldFirstBlock = getStartTree()->getNode()->getBlock();
+   TR::Node *glRegDeps=NULL;
+   if (oldBBStart->getNumChildren() == 1)
+      glRegDeps = oldBBStart->getChild(0);
+
+   TR::CFG *cfg = getFlowGraph();
+   TR::Compilation *comp = cfg->comp();
```

Comment:
Is this needed?  This is already in the `Compilation` class so `self()` should do.

Reply:
Fixed.